odl/solvers/functional/default_functionals.py

```diff
@@ -423,7 +423,7 @@ def __repr__(self):
 
 class IndicatorLpUnitBall(Functional):
 
-    """The indicator function on the unit ball in given the ``Lp`` norm.
+    r"""The indicator function on the unit ball in the :math:`L_p` norm.
```

Comment:
Avoid math in top comment, this should be readable (easily) without a fancy editor.